golangci.yaml

```diff
@@ -0,0 +1,24 @@
+linters:
```

Comment:
Added the linting configuration  for the tool and the ran it against one package to try it out.